### PR TITLE
Feat/homepage

### PR DESCRIPTION
**Pull request by karstenlenz**
Please review my code for the homepage component. It is just the simple homepage component without styling.

Story: https://github.com/karstenlenz/capstone-project/issues/2

Vercel-Link: https://capstone-project-hgr9cp14w.vercel.app/

### CODE REVIEW

File: src/common/Button.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+Button.propTypes = {
+  onClick: PropTypes.func,
+  children: PropTypes.string.isRequired,
+}
+
+export default function Button({ children, onClick }) {
+  return <BlockButton onClick={onClick}>{children}</BlockButton>
```
Comment (ninasander) [inline src/common/Button.js:11]: The name BlockButton might sound a little misleading. I know it references the display: block in your styling, but someone might assume this is a Button with a block function. Maybe ButtonStyled would be an easy alternative?

**Review by ninasander — COMMENTED** [review]
Good job! Just a minor thing I commented on :)

File: src/Homepage/__snapshots__/Homepage.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The Homepage component renders correctly 1`] = `
+<div>
+  <img
+    alt=""
```
Comment (slawoe) [inline src/Homepage/__snapshots__/Homepage.test.js.snap:6]: Maybe you forgot it or was it intention: there is no alt text

File: src/GlobalStyles.js [hunk 3]
```diff
@@ -18,27 +18,27 @@ main {
 
 
 h1 {
-    font-size: 2.5em;
+    font-size: 29px;
```
Comment (slawoe) [inline src/GlobalStyles.js:21]: wouldn't it be better in the old way to config the headings-size in em and not in px.
Do that you can define a font-size in the whole body and you don't have to change all of the headings-sizes afterwards manually, when you change font-size for the body, later?

File: src/Homepage/Homepage.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Button from '../common/Button'
+
+export default function () {
+  return (
+    <>
+      <img alt="" src="/img/mirror.svg" />
```
Comment (slawoe) [inline src/Homepage/Homepage.js:7]: Maybe you forgot it or was it intention: there is no alt text

File: src/GlobalStyles.js (hunk 3 above)
Comment (manuelpapa) [reply src/GlobalStyles.js:21]: > wouldn't it be better in the old way to config the headings-size in em and not in px.
> Do that you can define a font-size in the whole body and you don't have to change all of the headings-sizes afterwards manually, when you change font-size for the body, later?

good point though, this also provides a better performance on different devices and screensizes.

**Review by slawoe — APPROVED** [review]
found no big mistakes, just a missing alt text and I add a comment about sizing of your headings

File: src/common/Button.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+const BlockButton = styled.button`
+  display: block;
+  padding: 10px;
+  width: 100%;
+  font-size: 18px;
```
Comment (manuelpapa) [inline src/common/Button.js:18]: As mentioned before, try using `rem`-values instead to provide a better experience on different screen sizes.

File: src/Homepage/__snapshots__/Homepage.test.js.snap (hunk 2 above)
Comment (karstenlenz) [reply src/Homepage/__snapshots__/Homepage.test.js.snap:6]: This is intentional. The image is just there for decorative purposes. It doesn't need to be read by a screenreader.

File: src/GlobalStyles.js (hunk 3 above)
Comment (karstenlenz) [reply src/GlobalStyles.js]: You are right! At first I didn't like the "weird" numbers for the rem-values and was worried it wouldn't scale well. But now I replaced it with rem and it looks fine.

**Pull request merged**